packages/components/src/mobile/bottom-sheet/styles.native.scss

```diff
@@ -284,7 +284,15 @@
 
 // used in both light and dark modes
 .placeholderColor {
-	color: #87a6bc;
+	color: $gray;
```

Comment:
I noticed this color is already defined globally:
https://github.com/WordPress/gutenberg/blob/6f2d112399b007ccf33e913183e36446d841c9f7/packages/base-styles/_colors.native.scss#L41